README.md

```diff
@@ -8,6 +8,33 @@ A minimal terminal for TinyGo devices supporting 256-color ANSI escape codes.
 
 ## How to compile examples
 
+### basic
+
+Displays basic text. Runs on the PyPortal using hardware scrolling.
+
+```
+tinygo flash -target pyportal ./examples/basic
+```
+
+### basic-clue
+
+Displays basic text. Runs on the Clue using software scrolling.
+
+```
+tinygo flash -target clue ./examples/basic-clue
+```
+
```

Comment:
#7 may require some minor documentation changes.

Reply:
Understood. Let's get that one done first, and then I will modify this PR accordingly.

Reply:
Updated.